apps/src/aichat/views/AichatView.tsx

```diff
@@ -271,93 +268,95 @@ const AichatView: React.FunctionComponent<LabProps> = () => {
   }, [dispatch]);
 
   return (
-    <div id="aichat-lab" className={moduleStyles.aichatLab}>
```

Comment:
The only change here is wrapping the view in a context provider (use Hide Whitespace for a more accurate view).